addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1240,6 +1240,54 @@ registry.ImagesLazyLoading = publicWidget.Widget.extend({
     },
 });
 
+registry.BackgroundShape = publicWidget.Widget.extend({
+    selector: '.o_we_shape',
+
+    /**
+     * @override
+     */
+    start() {
+        this._onBackgroundShapeResize();
+        window.addEventListener('resize', this._onBackgroundShapeResize.bind(this));
+        return this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        window.removeEventListener('resize', this._onBackgroundShapeResize);
```

Comment:
That does not work, you don't give the same function to `removeEventListener` as you gave to `addEventListener`.